internal/pkg/agent/application/monitoring/server_test.go

```diff
@@ -92,32 +91,31 @@ func TestHTTPReloadEnableBehavior(t *testing.T) {
 
 	for _, testCase := range cases {
 		t.Run(testCase.name, func(t *testing.T) {
-			serverReloader, err := NewServer(logp.L(), api.Config{}, nil, nil, fakeCoordCfg, "linux", testCase.initConfig)
+			serverReloader, err := NewServer(logp.L(), nil, nil, fakeCoordCfg, testCase.initConfig)
 			require.NoError(t, err)
 
 			t.Logf("starting server...")
 			serverReloader.Start()
 			if testCase.httpOnAtInit {
-				waitOnReturnCode(t, http.StatusOK, "liveness", "?failon=failed", serverReloader)
+				waitOnReturnCode(t, http.StatusOK, "processes", "", serverReloader)
```

Comment:
Why did all the `/liveness`  calls become `/processes` in this case ? Was this an inappropriate use of `/liveness` endpoint ?